src/main/java/teammates/ui/pagedata/InstructorFeedbackResultsPageData.java

```diff
@@ -1202,6 +1213,33 @@ private void buildTableColumnHeaderForRecipientQuestionGiverView(List<ElementTag
         return responseRows;
     }
 
+    private void addCommentsToResponseRow(FeedbackResponseAttributes response, FeedbackQuestionAttributes question,
+            InstructorFeedbackResultsResponseRow responseRow) {
+        responseRow.setCommentsOnResponsesAllowed(true);
+        List<String> responseGiverRecipientList = new ArrayList<>(bundle.emailNameTable.keySet());
+        Collections.sort(responseGiverRecipientList);
+        String giverNameAndTeam = bundle.getNameForEmail(response.giver);
+        String recipientNameAndTeam = bundle.getNameForEmail(response.recipient);
+
+        String giverTeam = bundle.getTeamNameForEmail(response.giver);
+        String recipientTeam = bundle.getTeamNameForEmail(response.recipient);
+
+        giverNameAndTeam = bundle.appendTeamNameToName(giverNameAndTeam, giverTeam);
+        recipientNameAndTeam = bundle.appendTeamNameToName(recipientNameAndTeam, recipientTeam);
+
+        List<FeedbackResponseCommentRow> comments =
+                buildResponseComments(giverNameAndTeam, recipientNameAndTeam, question, response);
+        if (!comments.isEmpty()) {
+            responseRow.setCommentsOnResponses(comments);
+        }
+        Map<FeedbackParticipantType, Boolean> responseVisibilityMap = getResponseVisibilityMap(question);
+        FeedbackResponseCommentRow addCommentForm = buildFeedbackResponseCommentAddForm(question, response,
+                responseVisibilityMap, giverNameAndTeam, recipientNameAndTeam);
+        responseRow.setAddCommentButton(addCommentForm);
+        responseRow.setResponseRecipientIndex(responseGiverRecipientList.indexOf(response.recipient));
+        responseRow.setResponseGiverIndex(responseGiverRecipientList.indexOf(response.giver));
+    }
```

Comment:
@xpdavid I looked at implementation of `emailNameTable`. My concern here is in following case we donot put `response.giver` in the map. https://github.com/TEAMMATES/teammates/blob/7248edb70413d8133fd11382fdc255b639eb7a82/src/main/java/teammates/logic/core/FeedbackSessionsLogic.java#L2004-L2017

I don't know if the above condition is true ever but I put breakpoint on it and tested using sample sessions and I didn't find a case where giverType is team and response giver is a student's name.